ignite/cmd/model/chain_debug.go

```diff
@@ -74,7 +75,7 @@ func (m ChainDebug) View() string {
 		// that the output used by non UI based commands to keep
 		// consistency on error when the debug command is run
 		// without the `--server` flag.
-		return fmt.Sprintln(xstrings.ToUpperFirst(m.error.Error()))
+		return fmt.Sprintf("%s %s\n", icons.NotOK, xstrings.ToUpperFirst(m.error.Error()))
```

Comment:
I think we should decide for one consistent result for errors. Using the icon and potentially the color looks nicer, but at the moment the default is to print the error message [here](https://github.com/ignite/cli/blob/030bcd4a357f408c63d73b755625e0417f7545b7/ignite/cmd/ignite/main.go#L57) without icon and color.

The idea here is to keep consistency on errors between Bubbletea and the default, so they should match.

Reply:
Sure, I wasn't sure of that fix and was expecting your point of view :) 

Can fix that test with the proper way please ? Don't hesitate to push a commit in that PR.

Reply:
> Can fix that test with the proper way please ? Don't hesitate to push a commit in that PR.

Which test :) ?

Reply:
Oh, I understand now! I can fix the test, yes, if we agree to settle on using the error format proposed in this change for every CLI error. That would require to change [cmd/ignite/main.go here](https://github.com/ignite/cli/blob/030bcd4a357f408c63d73b755625e0417f7545b7/ignite/cmd/ignite/main.go#L57) to match.

@aljo242 what do you think, should we keep the error plain or add some formatting?

Reply:
Changeset dc096b7